manifests/2.7.0/opensearch-2.7.0.yml

```
@@ -123,7 +123,7 @@ components:
       - gradle:dependencies:opensearch.version
   - name: sql
     repository: https://github.com/opensearch-project/sql.git
-    ref: 2.x
+    ref: 2.7
```

Please enclose 2.7 in quotes.
```suggestion
    ref: '2.7'
```

---

done.